src/modules/home/HomePage.vue

```diff
@@ -113,14 +114,27 @@ const profileContent = ref<ProfileContent>({
   introMessage: '',
 });
 
-const callouts = ref<BasicCalloutData[]>([]);
+const callouts = ref<GetBasicCalloutData[]>([]);
 
 // This page is behind auth so currentUser can't be null
 // TODO: is there a nicer way to handle this?
 const user = currentUser as Ref<GetMemberData>;
 
 onBeforeMount(async () => {
   profileContent.value = await fetchProfileContent();
-  callouts.value = await fetchCallouts();
+  callouts.value = (
+    await fetchCallouts({
+      rules: {
+        condition: 'AND',
+        rules: [
+          {
+            field: 'status',
+            operator: 'equal',
+            value: CalloutStatus.Open,
```

Comment:
Sweet 👍

Reply:
It's a bit verbose at the moment but the general idea is to use a standard query language for all resources on the API (where it makes sense). I'm interested if you have any thoughts on this, I looked around to see what sort of standards there were but couldn't see anything, although I found [FIQL](https://fiql-parser.readthedocs.io/en/stable/usage.html) interesting